tests/cli/functional/actions/test_auth_actions.py

```diff
@@ -128,8 +116,11 @@ def test_unlock_nucypher_keyring_invalid_password(mocker, test_emitter, stdout_t
                                 character_configuration=alice_blockchain_test_config)
     keyring_attach_spy.assert_called_once()
 
+    captured = capsys.readouterr()
+    assert "Decrypting alice keyring..." in captured.out
```

Comment:
Can you use `DECRYPTING_CHARACTER_KEYRING` here with the {name} provided? Lower down in ` test_unlock_nucypher_keyring_dev_mode` you used

```suggestion
    message = DECRYPTING_CHARACTER_KEYRING.format(name=alice_blockchain_test_config.NAME)
    assert message in captured.out
```

Reply:
Fixed.